### DISCUSSION TITLE
Linkage of eForms with old TED notices.

### DISCUSSION THREAD

**Discussion by maximAtanasov** [OP-TED/eForms-SDK#436] [Q&A] [answered]
Hello,

As we are currently working on the migration of TED Procedures to eForms, we have a few question regarding the linking of notices. In particular, I would like to clarify the usage of BT-125, BT-758 and OPP-090. It seems that linkage between different notices of the same procedure and of changes for a single notice is not to be achieved only with OPP-090 and many different possibilities are available. Therefoce, I would like to discuss a few different migration scenarios:

1. Update of a published Prior Information Notice

In this scenario, an existing Prior Information Notice in TED-XML format (F01), that has already been published by TED, must be updated using an eForms change notice. As OPP-090 seems to be forbidden for planning notices, we have the possibility of using BT-125 or BT-758. However, according to the technical specifications of eForms, BT-758 is only allowed to have values of type UUID, which excludes notice publication IDs (XXXXXX-YYYY). Could you please clarify, exactly how BT-758 and BT-125 are to be used in this case or if there is any other possibility of linking the change notice to the original TED form?

2. Creation of a Contract Notice from a Prior Information Notice

In this scenario, a new eForms Contract Notice is created from an existing Prior Information Notice in TED-XML format (F01). Since we are not dealing with a change notice in this case, but rather a completely new one, we can only consider BT-125 and OPP-090. Could you please clarify which Bussiness Term should receive the notice publication ID of the prior information notice in order to achieve the linking?

3. Update of a published Contract Notice

In this scenario, an existing Contract Notice in TED-XML format (F02), must be updated using an eForms change notice. Unlike for procedures of type planning, however, OPP-090 is allowed in competition noitices. However, there is still the possibility of using BT-758 and BT-125 in order to perform linking. At the same time, a link to the prior information notice should also be possible. Could you please clarify what values BT-758 and OPP-090 should receive in this case?

4. Creation of a Contract Award Notice (or Ex-Ante transparency Notice) from an existing Contract Notice

In this scenario, an new eForms Contract Award Notice is created from an existing Contract Notice in TED-XML format (F02). Once again, we have the possibility of using BT-125 and OPP-090. The SDK documentation of BT-125 is not completely clear on whether its use is reserved for linkage to notices of type planning or if notices of type competition are also allowed. The current version of the TED-XML converter (0.6.0) maps the reference to the contract notice, that is contained in the award notice XML (F03) to BT-125. Comments in the code suggest that BT-125 attains different meanings depending on its position in the form (notice or lot). Could you please clarify what exactly is meant by this and if OPP-090 should be used instead in this case?

5. Update of a published Contract Award Notice (or Ex-Ante transparency Notice)

In this scenario, an existing Contract Award Noitce in TED-XMl format (F03), must be updated using an eForms change notice. Here too, we have the same issues as in scenario 3. A way to link to the original contract notice as well as the initial award notice (F03) must be available. This is, once again, like achieved with a combination of BT-758, BT-125 and OPP-090. Could please provide details on the exact usages of these bussiness terms?

Thank you for your time!

**Comment by rouschr** [comment]
This could be a question for @pdonohoe

**Comment by pdonohoe** [comment, marked as answer]
Dear @maximAtanasov Please could you ask this question via the Helpdesk: https://simap.ted.europa.eu/contact ?
Many thanks
